.pre-commit-config.yaml

```diff
@@ -4,12 +4,15 @@ repos:
     hooks:
     - id: trailing-whitespace
       exclude: test_data
+      exclude_types: ["diff"]
```

Comment:
the changes in this file are not needed anymore as I removed all the patch files.

Maybe its useful to keep this here anyway, up to you